Tasks/AzureDevOpsTestPlanV0/runTestPlan.ts

```diff
@@ -0,0 +1,18 @@
+import * as tl from 'azure-pipelines-task-lib/task';
+import { manualTestsFlow} from './manualTests'
+import { automatedTestsFlow} from './automatedTests'
+
+export async function run() {
+
+    const testSelectorInput = tl.getInput('testSelector');
+    console.log('Test Selector selected : ' + testSelectorInput);
+
+    if (testSelectorInput === 'manualTests') {
+        manualTestsFlow();
+    }
+    if (testSelectorInput === 'automatedTests') {
```

Comment:
This check is not required. As far as I understand, the selector input will be either manual or automated, since it is picklist.

Reply:
User can select both automated and manual or either of these 2 flows, hence we require this check. Let me know if you have any other thoughts on this.

Reply:
updated the condition